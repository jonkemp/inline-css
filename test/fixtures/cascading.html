<html><link rel="stylesheet" href="cascading.css"/><body>
<ul>
  <li>
    <ul>
      <li>
        <ul>
          <li></li>
        </ul>
      </li>
    </ul>
  </li>
</ul></body></html>
