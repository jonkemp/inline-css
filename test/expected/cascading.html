<html><head></head><body>
<ul>
  <li style="color: #f00; display: block;">
    <ul>
      <li style="color: #f00; display: block;">
        <ul>
          <li style="color: #f00; display: block;"></li>
        </ul>
      </li>
    </ul>
  </li>
</ul>
</body></html>
